source/common/http/http1/codec_impl.cc

```diff
@@ -485,6 +514,10 @@ void ConnectionImpl::onHeaderValue(const char* data, size_t length) {
   header_parsing_state_ = HeaderParsingState::Value;
   current_header_value_.append(data, length);
 
+  validateHeaderMapSize();
```

Comment:
Why was this split into another function?

Reply:
For some reason I was thinking I needed to reuse this function. Turns out that was not the case. I'll revert it.

Reply:
For some reason I was thinking I needed to reuse this function. Turns out that was not the case. I'll revert it.